securesystemslib/metadata.py

```diff
@@ -94,3 +90,78 @@ def pae(self) -> bytes:
             len(self.payload),
             self.payload,
         )
+
+    def sign(self, signer: Signer) -> Signature:
+        """Sign the payload and create the signature.
+
+        Arguments:
+            signer: A "Signer" class instance.
+
+        Note:
+            DSSE signatures created have mandatory keyid due to underlying sign
+            methods (Issue #416) and is not according to DSSE specification.
+
+        Returns:
+            A "Signature" instance.
+        """
+
+        signature = signer.sign(self.pae())
+        self.signatures.append(signature)
+
+        return signature
+
+    def verify(self, keys: List[Key], threshold: int) -> Dict[str, Key]:
+        """Verify the payload with the provided Keys.
+
+        Arguments:
+            keys: A list of public keys to verify the signatures.
+            threshold: Number of signatures needed to pass the verification.
+
+        Raises:
+            ValueError: If "threshold" is not valid.
+            SignatureVerificationError: If the enclosed signatures do not pass
+                the verification.
+
+        Note:
+            Mandating keyid in signatures and matching them with keyid of Key
+            in order to consider them for verification, is not a DSSE spec
+            complaint (Issue #416).
```

Comment:
complaint != compliant

```suggestion
            in order to consider them for verification, is not DSSE spec
            compliant (Issue #416).
```

Reply:
"compliant" is an adjective, hence my suggestion to also remove the article "a"